cmd/utils/flags.go

```diff
@@ -1571,6 +1593,60 @@ func setWhitelist(ctx *cli.Context, cfg *ethconfig.Config) {
 	}
 }
 
+func setMEV(ctx *cli.Context, ks *keystore.KeyStore, cfg *miner.Config) {
```

Comment:
As some validator are running in a private network and being protected by a sentry node, is there any solution for them?

Reply:
Yes, there is another branch for the sentry node to be a proxy between the MEV relays and the validator